Cloudflare Zero Trust Event Collector (new) (#37861)
demisto/content: Packs/CloudflareZeroTrust/Integrations/CloudflareZeroTrustEventCollector/CloudflareZeroTrustEventCollector_description.md

## Cloudflare Zero Trust

_____
Use this integration to fetch account audit logs, user audit logs, and access authentication logs from Cloudflare Zero Trust as events in Cortex XSIAM.
_____

#### **1. Retrieve Global API Key**

> - Visit [Retrieve Global API Key](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).

Reviewer: Why do I need to visit this page?
Maybe start with "To find how to retrieve a Cloudflare Global API Key, see [Retrieve Global API Key](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).

  **Note:** The Global API Key is an all-purpose token that can read and edit any data or settings that you can access in the dashboard.  

>   - API Key is available here: [Cloudflare API Tokens](https://dash.cloudflare.com/profile/api-tokens)  

Reviewer: Is the API Key different than the Global API Key?
I find this very confusing.
Why do I need to go here?

#### **2. Find Associated Email**

- Visit [Verify Email Address](https://developers.cloudflare.com/fundamentals/setup/account/verify-email-address/).

> - **API email** can be found here: [Cloudflare Profile](https://dash.cloudflare.com/profile).  

Reviewer: Again, need more information.  This is very confusing to me.

## Cloudflare Zero Trust

Use this integration to fetch account audit logs, user audit logs, and access authentication logs from Cloudflare Zero Trust as events in Cortex XSIAM.

#### **Retrieve Global API Key**

> - Go to [Cloudflare API Tokens](https://dash.cloudflare.com/profile/api-tokens).

Reviewer: ```suggestion
1. Go to [Cloudflare API Tokens](https://dash.cloudflare.com/profile/api-tokens).
```

> - Scroll to the API Keys section.

Reviewer: ```suggestion
2. Scroll to the **API Keys** section.
```

> - Here, you will find the Global API Key. Click View to reveal it (you may need to enter your password).

Reviewer: ```suggestion
3. Click **View** to reveal the Global API Key.  (You may need to enter your password).
```

> - To learn more about the Global API Key, visit the [Cloudflare API Key Documentation.](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).

Reviewer: ```suggestion
To learn more about the Global API Key, visit the [Cloudflare API Key Documentation.](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).
```

#### **Find Associated Email**

> - API email can be found here: [Cloudflare Profile](https://dash.cloudflare.com/profile).

Reviewer: ```suggestion
The associated API email can be found here: [Cloudflare Profile](https://dash.cloudflare.com/profile).
```

> - For more details on verifying your associated email, visit the  [Verify Email Address](https://developers.cloudflare.com/fundamentals/setup/account/verify-email-address/).

Reviewer: ```suggestion
For more details on verifying your associated email, visit the  [Verify Email Address](https://developers.cloudflare.com/fundamentals/setup/account/verify-email-address/).
```

#### **Token Permissions**

> This API token will affect the below accounts and zones, along with their respective permissions:

Reviewer: ```suggestion
This API token will affect the following accounts and zones, along with their respective permissions:
```
